scss/main.scss

```diff
@@ -225,3 +225,31 @@ h3 {
         display: inline;
     }
 }
+
+.page-header {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.page-header .filter {
+    margin-bottom: 11px;
+    align-self: center;
+}
+
+.filter {
+    *:not(:last-child) {
+        margin-right: 15px;
+    }
+
+    input,
+    select {
+        padding: 6px 12px;
+        box-sizing: border-box;
+        height: 38px;
+    }
+
+    input {
+        width: 250px;
+    }
+}
```

Comment:
will need to caniuse for:

- flexbox
- justify-content: space-between
- :not(...)
- :last-child (though iirc this has wide support)

Reply:
i was able to get rid of all those items as i moved the inputs out of the header.